fragments/platform/rhel8/repos/default.ks

```diff
@@ -0,0 +1,3 @@
+# Default Fedora Rawhide repositories
+url BASE_OS_REPO_URL_GOES_HERE
+repo --name=appstream APPSTREAM_REPO_URL_GOES_HERE
```

Comment:
We can do this or we can use `unified` solution. The second sounds easier to me.

Reply:
Sounds, good, I'll add one more comment line.